TTCN3_Project/CANFilter/com/testingtech/playits/canfilter/tests/CANFilterTest.java

```diff
@@ -0,0 +1,48 @@
+package com.testingtech.playits.canfilter.tests;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Hashtable;
+import java.util.List;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.testingtech.playits.canfilter.Car2XEntry;
+
+@RunWith(Parameterized.class)
+public class CANFilterTest {
```

Comment:
Does this test class actually work or is still needed? I don't see any test methods with asserts and that fancy stuff.